### PR TITLE
Playback control of nested components and getting Display info

### PR DESCRIPTION
**Pull request by donnapep**
The `rise-page` web component has the following functionality:
- Listens for `rise-component-ready` and `rise-component-done` events from its child components.
- Controls the playback of its child components.
- Returns information about the Display.

### CODE REVIEW

**Comment by donnapep** [comment]
@stulees Please review. I will update the content of the ReadMe in a separate PR. Thx.

File: test/rise-page-ready.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+  <title>Page - Ready</title>
+
+  <script src="../../webcomponentsjs/webcomponents.min.js"></script>
+  <script src="../../web-component-tester/browser.js"></script>
+  <script>
+    // This needs to run before any timers are set.
+    var clock = sinon.useFakeTimers();
```
Comment (stulees) [inline test/rise-page-ready.html:12]: @donnapep Is this the solution to our previous frustrations using fakeTimers?

**Comment by stulees** [comment]
@donnapep :+1:

File: test/rise-page-ready.html (hunk 1 above)
Comment (donnapep) [inline test/rise-page-ready.html:12]: @stulees I'm hoping so. The symptoms were the same in that the timer was not firing until I moved this up here. We should try this out next time we're in those tests.

**Pull request merged**